web/tests/stream_list_sort.test.cjs

```diff
@@ -193,13 +176,7 @@ test("basics", () => {
 
     // Test searching part of word
     sorted_sections = sort_groups("tortoise").sections;
-<<<<<<< HEAD
-    assert.deepEqual(sorted_sections.length, 3);
-||||||| parent of e3f8b5fa9f (left_sidebar: Remove inactive section, put inactive channels in regular section.)
-        assert.deepEqual(sorted_sections.length, 3);
-=======
-        assert.deepEqual(sorted_sections.length, 2);
->>>>>>> e3f8b5fa9f (left_sidebar: Remove inactive section, put inactive channels in regular section.)
```

Comment:
Not ideal to have this in the commits, but probably not worth going back to fix it?